shared/util/feature-flags.native.js

```diff
@@ -5,7 +5,8 @@ import type {FeatureFlags} from './feature-flags'
 const ff: FeatureFlags = {
   admin: __DEV__,
   explodingMessagesEnabled: true,
-  fsEnabled: __DEV__,
+  fsEnabled: true,
```

Comment:
I'm realizing this wasn't turned on when we made M1 release. But luckily we don't check for this for native (at least now), so it didn't matter.